### DISCUSSION TITLE
Constantly Receiving a Chat Error

### DISCUSSION THREAD

**Discussion by ryanslam** [Soulter/hugging-chat-api#235] [Q&A]
Hey everyone! I've been trying to run my chat bot using the hugging-chat-api. It was working yesterday but when I ran it today, I kept receiving the ChatError Exception. The chat error id is: 07de222b-a3b2-4c17-b234-9e7f4e604b84. I was wondering what the cause behind this is?

**Comment by Soulter** [comment]
Hi, We have fixed the error, please update to the latest version v0.4.8
